package.json

```diff
@@ -11,41 +11,38 @@
     "lint": "lint-all-the-things",
     "start": "ember server",
     "test": "npm run lint && npm run test-addon",
-    "test-addon": "EMBER_TRY_SCENARIO=${EMBER_TRY_SCENARIO:=default} && ember try:one $EMBER_TRY_SCENARIO --- COVERAGE=true ember test"
+    "test-addon": "EMBER_TRY_SCENARIO=${EMBER_TRY_SCENARIO:=ember-default} && ember try:one $EMBER_TRY_SCENARIO --- COVERAGE=true ember test"
   },
   "repository": "git@github.com:ciena-frost/ember-frost-buckets.git",
   "engines": {
-    "node": ">= 5.0.0"
+    "node": ">= 6.0.0"
   },
   "author": "Andres Gonzalez (https://github.com/agonza40)",
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
+    "broccoli-asset-rev": "^2.4.5",
     "chai-jquery": "^2.0.0",
-    "ember-ajax": "0.7.1",
-    "ember-browserify": "1.1.13",
-    "ember-cli": "2.5.0",
-    "ember-cli-app-version": "^1.0.0",
+    "ember-ajax": "^2.4.1",
+    "ember-cli": "2.12.3",
     "ember-cli-chai": "0.3.2",
-    "ember-cli-code-coverage": "0.3.5",
-    "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mocha": "0.13.1",
-    "ember-cli-release": "0.2.8",
-    "ember-cli-sass": "6.1.0",
+    "ember-cli-code-coverage": "0.3.12",
+    "ember-cli-dependency-checker": "^1.3.0",
+    "ember-cli-htmlbars-inline-precompile": "^0.3.6",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-mocha": "0.13.2",
+    "ember-cli-shims": "^1.0.2",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-computed-decorators": "0.2.2",
-    "ember-concurrency": "0.7.17",
-    "ember-data": "^2.5.0",
+    "ember-computed-decorators": "0.3.0",
+    "ember-concurrency": "~0.7.17",
```

Comment:
Related to my other comment about why we are bumping versions of dependencies unrelated to the `ember-cli` upgrade, we are choosing to float this one, but not other ones, and how does a consumer even get this package for use?  This is a dev dependency so it will never be installed and there's not a blueprint installing it on their behalf as well.  So a consumer running `ember install ember-frost-buckets` will SOL.  But again, these are all discussions and changes for a different PR....